recipes/effolkronium_random/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8.11)
```

```suggestion
cmake_minimum_required(VERSION 3.1)
```

https://github.com/conan-io/conan-center-index/issues/2184